imqsauth/http.go

```diff
@@ -597,14 +640,20 @@ func httpHandlerUpdateUser(central *ImqsCentral, w http.ResponseWriter, r *httpR
 	firstname := strings.TrimSpace(r.http.URL.Query().Get("firstname"))
 	lastname := strings.TrimSpace(r.http.URL.Query().Get("lastname"))
 	mobilenumber := strings.TrimSpace(r.http.URL.Query().Get("mobilenumber"))
+	strAuthusertype := strings.TrimSpace(r.http.URL.Query().Get("authusertype"))
+	iAuthUserType, err := strconv.Atoi(strAuthusertype)
+	authUserType := authaus.AuthUserType(iAuthUserType)
```

Comment:
Now that I see the API in words, in the unit test, I think it would be better if the API accepted words, instead of numbers, for the AuthUserType. eg - "BuiltIn" or "LDAP". Rather than expose your internal codes to the API, use plain english words.